xtevent/xtevent.sthlp

```diff
@@ -209,9 +215,17 @@ option requires {help reghdfe} and {help ftools} to be installed. For IV estimat
 Note that standard errors may be different and singleton clusters may be dropped using {help reghdfe}. See Correia (2017).
 
 {phang}
-{opt addabsorb(varlist)} specifies additional fixed effects to be absorbed when using {help reghdfe}. By default, {cmd xtevent} includes time and
+{opt addabsorb(varlist)} specifies additional fixed effects to be absorbed when using {help reghdfe}. By default, {cmd:xtevent} includes time and
 unit fixed effects. {opt addabsorb} requires {opt reghdfe}.
 
+{phang}
+{opt cohort(varname)} specifies the variable that identifies the cohort for each unit. {opt cohort} and {opt control_cohort} indicates {cmd:xtevent} to estimate the even-time coefficients with the Interaction Weighted Estimator
+ proposed by Sun and Abraham (2021).
+
+{phang}
+{opt control_cohort(varname)} specifies the binary variable that identifies the control_cohort. {opt cohort} and {opt control_cohort} indicates {cmd:xtevent} to estimate the even-time coefficients with the Interaction Weighted Estimator

```

Comment:
@Constantino-Carreto-Romero "... the binary variable that identifies the control cohort" (no underscore)